tests/reg_tests/test_DVGeometry.py

```diff
@@ -805,7 +805,74 @@ def test_22(self, train=False, refDeriv=False):
             handler.par_add_norm('norm', norm_diff, rtol=1e-7, atol=1e-7)
             os.remove(copyName)
 
-            
+    def train_23_xyzFraction(self, train=True):
+        self.test_23_xyzFraction(train=train)
+
+    def test_23_xyzFraction(self, train=False):
+        """
+        Test 23
```

Comment:
Can you add a description of what it tests

Reply:
yes, will add the description from the PR descr above